packages/react-components/react-combobox/src/components/Dropdown/useDropdown.tsx

```diff
@@ -104,6 +104,7 @@ export const useDropdown_unstable = (props: DropdownProps, ref: React.Ref<HTMLBu
   triggerSlot = slot.always(props.button, {
     defaultProps: {
       type: 'button',
+      tabIndex: 0,
```

Comment:
nit: Maybe add a comment why this is needed, with a link to the issue.

Reply:
added a comment! There's no issue to link to, since this is Safari's explicit interaction decision, and they don't consider it a bug :/